com.Acestream.engine.json

```diff
@@ -0,0 +1,149 @@
+{
+    "app-id": "com.Acestream.engine",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.36",
+    "sdk": "org.gnome.Sdk",
+    "command": "acestream",
+    "tags": [
+        "proprietary"
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home"
```

Comment:
Why does it need full home access? It's better to have proprietary, network-connected app limited to specific directory such as `xdg-download and `xdg-videos` rather than let it access everything by default.

Reply:
I checked it on the local snapshot - yes, host is enough.